### PR TITLE
Update doctrine.rst

### PR DESCRIPTION
**Pull request by atailouloute**

### CODE REVIEW

File: doctrine.rst [hunk 1]
```diff
@@ -539,17 +539,17 @@ Take a look at the previous example in more detail:
...
-* **lines 15-18** In this section, you instantiate and work with the ``$product``
+* **lines 14-17** In this section, you instantiate and work with the ``$product``
   object like any other normal PHP object.
 
-* **line 21** The ``persist($product)`` call tells Doctrine to "manage" the
+* **line 20** The ``persist($product)`` call tells Doctrine to "manage" the
   ``$product`` object. This does **not** cause a query to be made to the database.
 
-* **line 24** When the ``flush()`` method is called, Doctrine looks through
+* **line 22** When the ``flush()`` method is called, Doctrine looks through
```
Comment (wouterj) [inline doctrine.rst]: Ah, one mistake: This should have bee 23 :)

**Review by wouterj — APPROVED** [review]
Nice catch!

There was one little mistake, but we can fix it during the merge. :+1:

File: doctrine.rst (hunk 1 above)
Comment (atailouloute) [reply doctrine.rst]: @wouterj you're right, I fixed it

**Comment by xabbuh** [comment]
Thank you @atailouloute.

**Pull request closed without merging**